azure-pipelines.yaml

```diff
@@ -19,6 +21,15 @@ trigger:
     include:
       - dev
       - master
+      
+schedules:
```

Comment:
Why do we need scheduled build for dev? Since master has the latest production code - Will that be enough?

Reply:
yeah, it should be enough ideally, but thought good to have it running for dev also. can remove for dev

Reply:
updated